stations.csv

```diff
@@ -62013,3 +62013,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 66144;Örnsköldsvik Fjällräven Center;ornskoldsvik-fjallraven-center;;;63.28417;18.72444;35819;SE;Europe/Stockholm;f;f;f;f;t;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66144;
 66145;Östersund Central;ostersund-central;;;63.17028;14.63778;35768;SE;Europe/Stockholm;f;f;f;f;t;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;エステルスンド;외스테르순드;;;Эстерсунд;;;厄斯特松德;urn:trainline:public:nloc:csv66145;
 66146;Cassino Abbazia;cassino-abbazia;8313956;;41.2924;13.4850;19827;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8313956;f;;f;;f;;f;;f;;f;;f;;f;8313956;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;カッシーノ;;;;Кассино;;;;urn:trainline:public:nloc:csv66146;
+66147;Boulazac Isle Manoire;boulazac-isle-manoire;8772547;87725473;45.172096;0.772803;;FR;Europe/Paris;t;f;f;t;f;f;FRJTO;;t;;f;8772547;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv66147;
```

Comment:
Boulazac Isle Manoire in this context is a train stop, not a city, so `is_city` should be false.